libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -666,9 +666,9 @@ inline void JSONNode::set_name(const json_string & newname) json_nothrow{
 	   return static_cast<json_string>(*internal);
     }
 
-    inline int JSONNode::as_int(void) const json_nothrow {
+    inline json_int_t JSONNode::as_int(void) const json_nothrow {
```

Comment:
`json_int_t` is the type defined in the options in `JSON_INT_TYPE`, currently `long long`, which is compatible with (actually the very same as) `std::streamoff`.